vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -38,6 +40,16 @@ func (oa OpenAPI) Install(c *restful.Container, mux *mux.PathRecorderMux) (*hand
 	if err != nil {
 		klog.Fatalf("Failed to register open api spec for root: %v", err)
 	}
+
+	// we shadow ClustResourceQuotas with a CRD. This loop removes all CRQ paths
+	// from the OpenAPI spec such that they don't conflict with the CRD
+	// apiextensions-apiserver spec during merging.
+	for pth := range spec.Paths.Paths {
+		if strings.HasPrefix(pth, "/apis/quota.openshift.io/v1/clusterresourcequotas") {
```

Comment:
why not expose the list from our other carry patch here? so people don't have to maintain two lists